vizro-core/src/vizro/_vizro.py

```diff
@@ -173,13 +163,41 @@ def _reset():
         dash._pages.CONFIG.clear()
         dash._pages.CONFIG.__dict__.clear()
 
-    @staticmethod
-    def _get_external_assets(folder: Path, extension: str) -> List[str]:
-        """Returns a list of paths to assets with given `extension` in `folder`, prefixed with `STATIC_URL_PREFIX`.
 
-        e.g. with STATIC_URL_PREFIX="vizro", extension="css", folder="/path/to/vizro/vizro-core/src/vizro/static",
-        we will get ["vizro/css/accordion.css", "vizro/css/button.css", ...].
-        """
-        return sorted(
-            (STATIC_URL_PREFIX / path.relative_to(folder)).as_posix() for path in folder.rglob(f"*.{extension}")
+def _make_resource_spec(path: Path):
+    # For dev versions, a branch or tag called e.g. 0.1.20.dev0 does not exist and so won't work with the CDN. We point
+    # to main instead, but this can be manually overridden to the current feature branch name if required.
+
+    # _git_branch = __version__ if "dev" not in __version__ else "main"
+    _git_branch = "feat/allow-servering-external-assets"
```

Comment:
```suggestion
    _git_branch = __version__ if "dev" not in __version__ else "main"
```
Must be applied before merging

Reply:
Ah yes, will do! The time delay is ok, it's just something to do with the CDN content caching I guess. If it causes problems with testing then it's possible to clear it manually: https://www.jsdelivr.com/tools/purge.

btw we should only very rarely need to set `_git_branch` manually here. It would really only be case where for some reason you need to test with `serve_locally=False` AND you've changed some of the assets. Apart from PRs like this that won't ever really happen.